sql-statements/sql-statement-show-stats-healthy.md

```
@@ -87,5 +87,5 @@ mysql> SHOW STATS_HEALTHY;
 
 ## 参照 {#see-also}
 
--   [ANALYZE](/sql-statements/sql-statement-analyze-table.md)
+-   [ANALYZE TABLE](/sql-statements/sql-statement-analyze-table.md)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)